tests/test_DGEList.py

```diff
@@ -199,15 +193,59 @@ def test_cpm():
     first_pos = dge_list.counts[0][0]
     col_sum = np.sum(dge_list.counts, axis=0)
     assert isinstance(first_pos, np.integer)
-    dge_list.cpm()
-    assert dge_list.counts[0][0] == first_pos * 1e6 / col_sum[0]
+    new_dge_list = dge_list.cpm()
+    assert new_dge_list.counts[0][0] == first_pos * 1e6 / col_sum[0]
+
+
+def test_rpkm():
+    dge_list = DGEList(filename=str(get_dataset_path(TEST_DATASET_NPZ)))
+    icd = ImportCanonicalData(
+        get_dataset_path(TEST_GENE_SET_DATA), get_dataset_path(TEST_GENE_SYMBOLS)
+    )
+    first_pos = dge_list.counts[0][0]
+    first_gene = dge_list.genes[0]
+
+    col_sum = np.sum(dge_list.counts, axis=0)
+    assert isinstance(first_pos, np.integer)
+    rpm_dge = dge_list.rpkm(icd)
+    ensg_gene = icd.pick_gene_id(icd.get_genes_from_symbol(first_gene))
+    gene_len = icd.get_length_of_canonical_transcript(ensg_gene)
+    # RPKM=numReads / (geneLength / 1000 * totalNumReads / 1, 000, 000)
+    print(gene_len)
+    print(rpm_dge.counts[0][0])
+    print(first_pos)
+    print(first_pos / ((gene_len / 1e3) * (col_sum[0] / 1e6)))
```

Comment:
You are doing here what @clintval criticized in test_tpm before as you are transfering the logic from rpkm() to its test function (compare this comment).
https://github.com/r-bioinformatics/edgePy/pull/72#discussion_r224543989
We could be more consistent and use the simple dataset as in test_tpm().
Another option: 
- We use the TEST_DATASET_NPZ as is
- Using edgeR, we normalize that dataset and store the normalized count table
- In our test we compare to the stored normalized count table.
- In the docstring of cpm(), tpm() and rpkm() we can document the edgeR equivalent of these functions.
What do you think?